### PR TITLE
Benchmark Tool in Go

### PR DESCRIPTION
**Pull request by devoshm**
Create users.
Create functions.
Submit jobs to Open Whisk in parallel.
Log the time and analyze the parallelism.

### CODE REVIEW

File: ow-wrapper.go [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+	fmt.Println("Execution Completed.")
+}
+
+func execCmdsFromFile(filePath string, needCreation bool) {
+	fmt.Println("Parsing File: " + filePath)
+	fread, _ := os.Open(filePath)
+	scanner := bufio.NewScanner(fread)
+
+	timeVsUserFuncMap := make(map[int][]UserFuncs)
```
Comment (devoshm) [inline ow-wrapper.go:42]: Change the time into a timer to invoke the functions at the mentioned time.

File: ow-wrapper.go [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+		userAuth := cmdMap[USER_AUTH]
+		functionID := cmdMap[FUNCTION_ID]
+
+		start := time.Now()
+		execResult := execCmd([]string{"invokeFunction", functionID, userAuth})
+		elapsed := time.Since(start)
+
+		resultMap := copyMap(cmdMap)
+		delete(resultMap, USER_AUTH)
+		resultMap[CMD_RESULT] = execResult
```
Comment (devoshm) [inline ow-wrapper.go:171]: Split the command results into args given by server.

File: constants.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	USER_AUTH           = "UserAuth"
+	FUNCTION_ID         = "FunctionID"
+	SEQ                 = "Seq"
+	CMD_RESULT          = "CmdResult"
+	ELAPSED_TIME_IN_NS  = "ElapsedTimeInNS"
+	ELAPSED_TIME_IN_SEC = "ElapsedTimeInSec"
+	SUBMITTED_AT        = "StartTime"
+	ENDED_AT            = "EndTime"
+
+	OPEN_WHISK_CONCURRENCY_FACTOR = 4
```
Comment (devoshm) [inline constants.go:15]: Make it a command line argument

**Comment by jimcadden** [comment]
Thanks, Devosh. This looks great

**Pull request merged**